connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -91,6 +91,17 @@ message WriteOperation {
   // (Optional) A list of configuration options.
   map<string, string> options = 9;
 
+  message SaveTable {
+    string table_name = 1;
```

Comment:
Can you follow https://github.com/apache/spark/blob/master/connector/connect/docs/adding-proto-messages.md to add `(Required)` or `(Optional)` with explanation on the fields?